### PR TITLE
Remove `String.fromCodePoint` shim

### PR DESCRIPTION
**Pull request by mathiasbynens**
The `String.fromCodePoint` shim is not necessary anymore once we drop support for Node.js v0.10 and v0.12.

Ref. #243, https://github.com/babel/babel/issues/4315, https://github.com/babel/babel/milestone/9.

| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| Breaking change?  | yes
| New feature?      | no
| Deprecations?     | no
| Spec compliancy?  | yes
| Tests added/pass? | yes (in modern Node.js versions)
| Fixed tickets     | #243, https://github.com/babel/babel/issues/4315, https://github.com/babel/babel/milestone/9
| License           | MIT

### CODE REVIEW

**Comment by hzoo** [comment]
Ooh maybe we could use github projects across the org to track 7.x as well?

**Comment by mathiasbynens** [comment]
I was just wondering about that :) A shared milestone would be ♥

File: test/fixtures/jsx/basic/entity/expected.json [hunk 1]
```diff
@@ -136,12 +136,12 @@
                 }
               },
               "extra": null,
-              "value": "💩"
+              "value": "\uD83D\uDCA9"
```
Comment (mathiasbynens) [inline test/fixtures/jsx/basic/entity/expected.json]: Note: this doesn’t change the actual string value (`"\uD83D\uDCA9" === "💩"`), it just ensures the file is ASCII-safe.

**Comment by hzoo** [comment]
Wow and we can also use this now (make a 7.x branch)

screen shot 2017-01-10 at 12 28 27 pm

**Comment by danez** [comment]
Thanks, merged into 7.0 branch.

**Comment by hzoo** [comment]
Not sure how we want to do it but made https://github.com/orgs/babel/projects

**Pull request merged**